Description: Change sample preferences to reflect expected Debian paths.
 Change the paths in the example preferences file to reflect that expected
 on Debian systems. Change is vendor specific so was not forwarded upstream.
Author: Peter Karlsson <peterk@debian.org>
Last-Update: 2013-02-08

--- crashmail-0.74.orig/doc/example.prefs
+++ crashmail-0.74/doc/example.prefs
@@ -32,7 +32,7 @@ SYSOP "Johan Billing"
 ; In the Linux version of CrashMail, it is possible to enter "syslog" as the
 ; filename. If you do this, everything will be logged to the syslog instead.
 
-LOGFILE "c:\\fido\\logs\\crashmail.log"
+LOGFILE "/var/log/crashmail.log"
 LOGLEVEL 5
 
 ; DUPEFILE <filename> <maxnumber>
@@ -49,7 +49,7 @@ LOGLEVEL 5
 ;  KILL       Dupes are killed
 ;  IGNORE     No dupechecking
 
-DUPEFILE "c:\\fido\\logs\\crashmail.dupes" 200
+DUPEFILE "/var/log/crashmail.dupes" 200
 DUPEMODE BAD
 
 ; LOOPMODE IGNORE/LOG/LOG+BAD
@@ -97,41 +97,41 @@ DEFAULTZONE 2
 ; The inbound directory is the directory where CrashMail looks for .pkt files
 ; and bundles to toss.
 
-INBOUND "c:\\fido\\inbound"
+INBOUND "/var/spool/ftn/inb"
 
 ; OUTBOUND <directory>
 ;
 ; The outbound directory is where CrashMail writes the flow files that tells
 ; the mailer what files to send.
 
-OUTBOUND "c:\\fido\\outbound"
+OUTBOUND "/var/spool/ftn/outb"
 
 ; TEMPDIR <directory>
 ;
 ; This is the directory where CrashMail unpacks incoming bundles.
 ;
 
-TEMPDIR "c:\\fido\\temp"
+TEMPDIR "/tmp"
 
 ; CREATEPKTDIR <directory>
 ;
 ; This is the directory where CrashMail stores created .pkt files until they
 ; are stored in the packet directory.
 
-CREATEPKTDIR "c:\\fido\\temp"
+CREATEPKTDIR "/tmp"
 
 ; PACKETDIR <directory>
 ;
 ; This is the directory where CrashMail stores generated bundles.
 
-PACKETDIR "c:\\fido\\packets"
+PACKETDIR "/var/spool/ftn/outb"
 
 ; STATSFILE <filename>
 ;
 ; This is the file where CrashMail stores statistics about areas and nodes.
 ; You can display the contents of this file with CrashStats.
 
-STATSFILE "c:\\fido\\crashmail.stats"
+STATSFILE "/var/spool/ftn/data/crashmail.stats"
 
 ; BEFORETOSS <command>
 ;
@@ -388,8 +388,8 @@ FILEATTACH "2:200/207.5"
 ; the bundle matches the recog string, CrashMail uses that packer. ? can be
 ; used as a wildcard and you can use $xx to specify a hexadecimal number.
 
-PACKER "LHA" "c:\\fido\\bin\\lha a %a %f" "c:\\fido\\bin\\lha x %a" "??-lh?-"
-PACKER "ZIP" "c:\\fido\\bin\\pkzip %a %f" "c:\\fido\\bin\\pkunzip %a" "PK"
+PACKER "LHA" "/usr/bin/lha a %a %f" "/usr/bin/lha %a" "??-lh?-"
+PACKER "ZIP" "/usr/bin/zip -j %a %f" "/usr/bin/unzip -j %a" "PK"
 
 ; AKA
 ; ===
@@ -514,7 +514,7 @@ REMOTESYSOP "Johannes Nilsson"
 ;
 ; The file that is sent when a downlink issues a %HELP command.
 
-AREAFIXHELP "c:\\fido\\AreafixHelp.txt"
+AREAFIXHELP "/usr/share/crashmail/AreafixHelp.txt"
 
 ; AREAFIXMAXLINES <max>
 ;
@@ -541,7 +541,7 @@ AREAFIXNAME "CrashMail"
 ; available for forward-requests. GROUP specifies the group needed to be
 ; allowed to forward-requests areas in this list.
 
-AREALIST 2:200/100.0 "c:\\fido\\lists\\R20Desc.lst" GROUP A FORWARD DESC
+AREALIST 2:200/100.0 "/var/spool/ftn/R20Desc.lst" GROUP A FORWARD DESC
 
 ; Routing
 ; =======
@@ -696,18 +696,18 @@ JAM_MAXOPEN 5
 ;
 ; You must use one of these %-codes or the new path will not be unique.
 
-NETMAIL "NETMAIL" 2:200/108.7 JAM "c:\\fido\\areas\\NETMAIL"
+NETMAIL "NETMAIL" 2:200/108.7 JAM "/var/spool/ftn/msg/NETMAIL"
 
-AREA "BAD" 2:200/108.7 JAM "c:\\fido\\areas\\BAD"
+AREA "BAD" 2:200/108.7 JAM "/var/spool/ftn/msg/BAD"
 
-AREA "DEFAULT_A" 2:200/108.7 JAM "c:\\fido\\areas\\%8"
+AREA "DEFAULT_A" 2:200/108.7 JAM "/var/spool/ftn/msg/%a"
 
-AREA "R20_INTRESSE" 2:200/108.7 JAM "c:\\fido\\areas\\36124179"
+AREA "R20_INTRESSE" 2:200/108.7 JAM "/var/spool/ftn/msg/R20_INTRESSE"
 EXPORT %2:200/100.0
 DESCRIPTION "Intresseklubben"
 GROUP A
 
-AREA "R20_TRASHCAN" 2:200/108.7 JAM "c:\\fido\\areas\\3612417a"
+AREA "R20_TRASHCAN" 2:200/108.7 JAM "/var/spool/ftn/msg/R20_TRASHCAN"
 EXPORT %2:200/100.0
 DESCRIPTION "Soptunnan"
 GROUP A
